snapper/Hooks.cc

```diff
@@ -30,6 +35,28 @@ namespace snapper
 {
     using namespace std;
 
+    static bool
+    _plugins_filter_entries(unsigned char type, const char* name)
+    {
+	// must start with digit
+	if (*name >= '0' && *name <= '9')
+	    return true;
+	return false;
+    }
+
+    void
+    Hooks::run_scripts(const list<string>& args)
+    {
+	    SDir dir("/usr/lib/snapper/plugins");
+
+	    vector<string> scripts = dir.entries(_plugins_filter_entries);
+	    std::sort(scripts.begin(), scripts.end());
+	    for (vector<string>::iterator it = scripts.begin(); it != scripts.end(); ++it)
```

Comment:
You can use range-based-for.

Reply:
what's that? :) my c++ is rusty, I copied that style from other bits in the code